packages/docs/docs/contributors/get-started.md

```diff
@@ -0,0 +1,37 @@
+# Get started
+
+Contributions bring a diversity of ideas to the project,
+which enriches the project.
```

Comment:
is the github editor cutting off the document ruining the format?

Reply:
I'm following [semantic line break](https://sembr.org/) to make units of thought more distinguishable and easy to work with.